libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -536,7 +536,7 @@ class ClientContext
         tcp_sent(_pcb, NULL);
         tcp_recv(_pcb, NULL);
         tcp_err(_pcb, NULL);
-        _pcb = NULL;
+        _pcb = 0;
```

Comment:
_pcb is a pointer, so assignment was current with NULL.

Reply:
```
$ grep '_pcb =' ClientContext.h
            _pcb = 0;
            _pcb = 0;
        _pcb = NULL;
```
It is for coherency. 0 is c++ way for NULL (I prefer NULL myself). Should I change them all ?

Reply:
I didn't realize, but yes please.

Reply:
Oh, yes, for some reason I thought of ClientContext as C code. Nullptr please!